app/assets/stylesheets/petitions/views/shared/_petition-item.scss

```diff
@@ -16,6 +16,9 @@
   .panel-indent {
     margin: 0;
   }
+  p {
+    margin-bottom: 0;
```

Comment:
This probably effects a few different `.petition-item`s. Should be OK cause we have margin-bottom on the item itself, but worth cross checking.